docs/_quarto.yaml

```diff
@@ -124,7 +124,7 @@ quartodoc:
 
   sections:
     - title: Catalog
-      desc: PLACEHOLDER CATEGORY DESCRIPTION
+      desc: "Find and register private and public assets."
```

Comment:
@nickvigilante here's my try at descriptions for the categories. Yes? No?

Reply:
LGTM!